game/lib/game/equipment_and_crafting/equipment.py

```diff
@@ -0,0 +1,40 @@
+from lib.game.items.item import Item
+import pygame
+
+
+class Equipment:
+
+    __item_list = [None, None, None, None, None]
+    __background = pygame.transform.scale(pygame.image.load('config/assets/item_background.png'), (64, 64))
+    __marked_background = pygame.transform.scale(pygame.image.load('config/assets/marked_item_background.png'), (64, 64))
+    __marked_index = 0
+
+    def pick_up_item(self, new_item: Item):
+        try:
+            index = self.__item_list.index(None)
+            self.__item_list.insert(index, new_item)
```

Comment:
Wyjątki raczej są używane tylko do jakichś nieprzewidzianych sytuacji. To, że ekwipunek jest pełny, jest raczej okej, lepiej jest sprawdzić najpierw czy jest miejsce i wtedy robić inserta zamiast robić inserta i czekać na wyjątek